ext/html_tokenizer_ext/parser.c

```diff
@@ -360,17 +361,34 @@ static inline int rawtext_context(struct parser_t *parser)
       ctx == TOKENIZER_SCRIPT_DATA || ctx == TOKENIZER_PLAINTEXT);
 }
 
+static inline const char *find_next_newline(const char *buf, long unsigned int length)
+{
+  long unsigned int i;
+
+  for(i = 0; i < length; i++) {
+    if(*(buf + i) == '\n')
```

Comment:
If `find_next_newline` is actually needed, `*(buf + i)` could be simplified to `buf[i]`.